src/components/spreadsheet/spreadsheet.ts

```diff
@@ -260,6 +260,7 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
   closeSidePanel() {
     this.sidePanel.isOpen = false;
     this.focusGrid();
+    this.sidePanel.panelProps?.onCloseSidePanel();
```

Comment:
I've a doubt, should it be `this.sidePanel.panelProps?.onCloseSidePanel?();` instead ?